### PR TITLE
htx_test.py: libncurses5 package issue

### PR DESCRIPTION
**Pull request by shirishaganta1**
Changed that one to libncurses6

Signed-off-by: Shirisha Ganta <SHIRISHA.Ganta1@ibm.com>

### CODE REVIEW

**Comment by shirishaganta1** [comment]
Before Patch:
localhost:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic # avocado run htx_test.py
JOB ID     : c04fa64a11b1d851b2bb10e4a64b3cdb96559e70
JOB LOG    : /home/siri/avocado-fvt-wrapper/results/job-2021-02-24T10.28-c04fa64/job.log
 (1/3) htx_test.py:HtxTest.test_start:  CANCEL: Can not install libncurses5 (2.05 s)
 (2/3) htx_test.py:HtxTest.test_check:  CANCEL: MDT file mdt.mem not found due to config (0.41 s)
 (3/3) htx_test.py:HtxTest.test_stop:  CANCEL: MDT file mdt.mem not found due to config (0.41 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 3
JOB HTML   : /home/siri/avocado-fvt-wrapper/results/job-2021-02-24T10.28-c04fa64/results.html
JOB TIME   : 3.45 s
localhost:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic # 

After patch 
Sles15sp3:
localhost:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic # avocado run htx_test.py
JOB ID     : 9e915d5e86541541ef56829811bd917cb8ebabdb
JOB LOG    : /home/siri/avocado-fvt-wrapper/results/job-2021-02-24T11.21-9e915d5/job.log
 (1/3) htx_test.py:HtxTest.test_start: PASS (40.87 s)
 (2/3) htx_test.py:HtxTest.test_check: PASS (120.56 s)
 (3/3) htx_test.py:HtxTest.test_stop: PASS (5.82 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/siri/avocado-fvt-wrapper/results/job-2021-02-24T11.21-9e915d5/results.html
JOB TIME   : 167.84 s
localhost:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic # 

Sles12sp5:
linux-7ve8:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic # avocado run htx_test.py
JOB ID     : ee10da72c9910281cfb5a337b3ad41cd85b5c3af
JOB LOG    : /home/siri/avocado-fvt-wrapper/results/job-2021-02-25T01.11-ee10da7/job.log
 (1/3) htx_test.py:HtxTest.test_start: PASS (35.28 s)
 (2/3) htx_test.py:HtxTest.test_check: PASS (120.80 s)
 (3/3) htx_test.py:HtxTest.test_stop: PASS (5.99 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/siri/avocado-fvt-wrapper/results/job-2021-02-25T01.11-ee10da7/results.html
JOB TIME   : 162.97 s
linux-7ve8:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic #

File: generic/htx_test.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def setup_htx(self):
             packages.extend(['libncurses5', 'g++',
                              'ncurses-dev', 'libncurses-dev'])
         elif detected_distro.name == 'SuSE':
-            packages.extend(['libncurses5', 'gcc-c++', 'ncurses-devel', 'tar'])
+            packages.extend(['libncurses6', 'gcc-c++', 'ncurses-devel', 'tar'])
```
Comment (harish-24) [inline generic/htx_test.py:79]: `libncurses5` is still available in Sles 15SP3 under the legacy module. Also this change causes failures on Sles15sp2, so make sure you handle this if you still want to pursue this.

Comment (shirishaganta1) [reply generic/htx_test.py:79]: Harish I tested with Sles15sp2 also and it is working fine with libncurses6
localhost:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic # avocado run htx_test.py
JOB ID     : 5ba75e2ff61783ca263bacf61ef0572adec3b23e
JOB LOG    : /home/siri/avocado-fvt-wrapper/results/job-2021-02-25T16.20-5ba75e2/job.log
 (1/3) htx_test.py:HtxTest.test_start: PASS (53.02 s)
 (2/3) htx_test.py:HtxTest.test_check: PASS (121.72 s)
 (3/3) htx_test.py:HtxTest.test_stop: PASS (7.86 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/siri/avocado-fvt-wrapper/results/job-2021-02-25T16.20-5ba75e2/results.html
JOB TIME   : 183.07 s
localhost:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic #

Comment (harish-24) [reply generic/htx_test.py:79]: @shirishaganta Sorry, I must have misguided you with `Sles15sp2` but as a matter of fact I intended to handle other SLES version like Sles12 for instance.

Comment (shirishaganta1) [reply generic/htx_test.py:79]: on sles12sp5 also it worked fine
Sles12sp5:
linux-7ve8:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic # avocado run htx_test.py
JOB ID : ee10da72c9910281cfb5a337b3ad41cd85b5c3af
JOB LOG : /home/siri/avocado-fvt-wrapper/results/job-2021-02-25T01.11-ee10da7/job.log
(1/3) htx_test.py:HtxTest.test_start: PASS (35.28 s)
(2/3) htx_test.py:HtxTest.test_check: PASS (120.80 s)
(3/3) htx_test.py:HtxTest.test_stop: PASS (5.99 s)
RESULTS : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML : /home/siri/avocado-fvt-wrapper/results/job-2021-02-25T01.11-ee10da7/results.html
JOB TIME : 162.97 s
linux-7ve8:/home/siri/avocado-fvt-wrapper/tests/avocado-misc-tests/generic #

Comment (harish-24) [reply generic/htx_test.py:79]: Right, verified that the changes wont affect distros starting SLES12SP2 which provides the package **only** in the maintenance updates and not the GA's.

While this change is valid what is the need for this change considering the following?
> libncurses5 is still available in Sles 15SP3 under the legacy module

Comment (shirishaganta1) [reply generic/htx_test.py:79]: After configuring legacy module repo it is working fine. So closing this PR. Sorry for the incovenience

**Pull request closed without merging**